readme.md

```diff
@@ -1,7 +1,6 @@
 # Search bar for Airbnb
-Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://Vasyliy22.github.io/layout_search-bar-airbnb/)
+- [TEST REPORT LINK](https://Vasyliy22.github.io/layout_search-bar-airbnb/report/html_report/)
```

Comment:
please add this links in your PR description

Reply:
Can you explain please how I have to add link? because I don't know